lib/diagnostic.rb

```diff
@@ -5,36 +5,38 @@ class Diagnostic
   # Assume each entry in the list implements a length method that returns
   # a Fixnum.
   def longest(list)
-    # Your code goes here
+    list.select { |entry| entry.size.max }
```

Comment:
You need to use the spaceship operator since we're doing a comparison. 

`list.max { |a, b| a.length <=> b.length }`